pkgs/development/libraries/mesa/default.nix

```
@@ -13,6 +13,7 @@
 , withValgrind ? !stdenv.isDarwin && lib.meta.availableOn stdenv.hostPlatform valgrind-light, valgrind-light
 , enableGalliumNine ? stdenv.isLinux
 , enableOSMesa ? stdenv.isLinux
+, libclc
 }:
```

```suggestion
, enableOpenCL ? stdenv.isLinux && stdenv.isx86_64
}:
```

some libclc's deps (spirv compiler, ...) are broken on non-x86_64, so mesa will be broken on non-x86_64 if we merge this in present form

---

at least it breaks i686

---

Thanks for noticing. I added that condition.